resources/charts/kubernetes/values.yaml

```diff
@@ -0,0 +1,43 @@
+apiserver:
+  apiserverKey: 
+  apiserverCert:
+  serviceAccountPub: 
+  caCert: 
+  etcdClientCaCert: 
+  etcdClientCert: 
+  etcdClientKey: 
+  aggregationCaCert: 
+  aggregationClientCert: 
+  aggregationClientKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  cloudProvider: 
+  etcdServers: 
+  aggregationFlags: 
+  serviceCIDR: 10.0.0.0/24
+  trustedCertsDir: /usr/share/ca-certificates
+controllerManager:
+  serviceAccountKey: 
+  caCert: 
+  caKey: 
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  cloudProvider: 
+  serviceCIDR: 10.0.0.0/24
+  podCIDR: 10.0.0.0/16
+  controlPlaneReplicas: 1
+  trustedCertsDir: /usr/share/ca-certificates
+kubeProxy:
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  podCIDR: 10.0.0.0/16
+  trustedCertsDir: /usr/share/ca-certificates
+kubeScheduler:
+  image: k8s.gcr.io/hyperkube:v1.16.2
+  controlPlaneReplicas: 1
+kubeConfigInCluster:
+  server: 
+podCheckpointer:
+  image: kinvolk/pod-checkpointer:83e25e5968391b9eb342042c435d1b3eeddb2be1
+coredns:
+  clusterDomainSuffix: cluster.local
+  controlPlaneReplicas: 1
+  image: coredns/coredns:coredns-amd64
+  clusterIP: 10.0.0.0/24
```

Comment:
@invidian please verify the default values for clusterIP

Reply:
This is usually 10th IP from service CIDR. that would be `10.0.0.10` by default then.